docker/hive/start-hive.sh

```diff
@@ -7,6 +7,16 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
 DEFAULT_FS=${DEFAULT_FS:=file:///}
 DB_TYPE=${DB_TYPE:=mysql}
 
+cat >${HIVE_HOME}/conf/hive-log4j.properties <<EOL
+log4j.rootLogger=INFO, CONSOLE
+
+# Disable excessive reflection warnings - KAFKA-5229
+log4j.logger.org.reflections=ERROR
```

Comment:
Do we need this?

Reply:
@lyogev i copied it from cdc: https://github.com/YotpoLtd/change-data-capture